--- src/CMakeLists.txt.origin	2016-12-18 15:31:07.079763549 +0100
+++ src/CMakeLists.txt	2016-12-18 15:31:14.669488710 +0100
@@ -135,7 +135,6 @@
 endif()
 
 if(NOT APPLE)
-  install(FILES ${QTROOT}/resources/qtwebengine_devtools_resources.pak DESTINATION resources)
   install(DIRECTORY ${CONAN_WEB-CLIENT-TV_ROOT}/tv DESTINATION ${INSTALL_RESOURCE_DIR}/web-client)
   if(DEFINED CONAN_WEB-CLIENT-DESKTOP_ROOT)
     install(DIRECTORY ${CONAN_WEB-CLIENT-DESKTOP_ROOT}/desktop DESTINATION ${INSTALL_RESOURCE_DIR}/web-client)
